README.md

```diff
@@ -117,7 +117,7 @@ Whenever you change dependencies (adding, removing, or updating, either in `pack
 - [How to add a new translation to MetaMask](./docs/translating-guide.md)
 - [Publishing Guide](./docs/publishing.md)
 - [How to use the TREZOR emulator](./docs/trezor-emulator.md)
-- [Developing on MetaMask](./development/README.md) 
+- [Developing on MetaMask](./development/README.md)
```

Comment:
The changes in this file seem to be unintended or unnecessary, could you remove please?

Reply:
Somehow I managed to miss that #15281 was already made and merged so that's how this came about. Amended.